### ISSUE TITLE
Testinfra tests don't pass under Vagrant

### ISSUE THREAD

**Issue by dortegau** [idealista/kafka-role#26]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Checked that your issue isn't already filled: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aidealista
    * Checked that there is not already provided the described functionality

### Description

Testinfra tests don't pass under Vagrant but in Docker:

```bash
___________________ test_kafka_service[ansible://kafka3.vm] ____________________

File = <class 'testinfra.modules.base.GNUFile'>
Service = <class 'testinfra.modules.base.SystemdService'>
Socket = <class 'testinfra.modules.base.LinuxSocketSS'>
AnsibleVars = {u'java_implementation': u'openjdk', u'kafka_host_name': u'127.0.0.1', u'kafka_hosts': [{u'host': u'kafka1.vm', u'id': 1}, {u'host': u'kafka2.vm', u'id': 2}, {u'host': u'kafka3.vm', u'id': 3}], u'kafka_port': 9999, ...}
Hostname = 'kafka3.vm'

    def test_kafka_service(File, Service, Socket, AnsibleVars, Hostname):
        if "kafka" in Hostname:
            host = AnsibleVars["kafka_host_name"]
            port = AnsibleVars["kafka_port"]
            assert File("/lib/systemd/system/kafka.service").exists
            assert Service("kafka").is_enabled
            assert Service("kafka").is_running
>           assert Socket("tcp://" + host + ":" + str(port)).is_listening
E           AssertionError: assert False
E            +  where False = <socket tcp://127.0.0.1:9999>.is_listening
E            +    where <socket tcp://127.0.0.1:9999> = <class 'testinfra.modules.base.LinuxSocketSS'>(((('tcp://' + u'127.0.0.1') + ':') + '9999'))
E            +      where '9999' = str(9999)

tests/test_ansible.py:56: AssertionError 
```

Seems to be a problem with the network interfaces because Kafka instances cannot connect to Zookeeper:

```
[2018-02-27 14:35:07,551] FATAL Fatal error during KafkaServerStartable startup. Prepare to shutdown (kafka.server.KafkaServerStartable)
org.I0Itec.zkclient.exception.ZkTimeoutException: Unable to connect to zookeeper server 'zookeeper.vm:2181' with timeout of 6000 ms
	at org.I0Itec.zkclient.ZkClient.connect(ZkClient.java:1233)
	at org.I0Itec.zkclient.ZkClient.<init>(ZkClient.java:157)
	at org.I0Itec.zkclient.ZkClient.<init>(ZkClient.java:131)
	at kafka.utils.ZkUtils$.createZkClientAndConnection(ZkUtils.scala:106)
	at kafka.utils.ZkUtils$.apply(ZkUtils.scala:88)
	at kafka.server.KafkaServer.initZk(KafkaServer.scala:326)
	at kafka.server.KafkaServer.startup(KafkaServer.scala:187)
	at kafka.server.KafkaServerStartable.startup(KafkaServerStartable.scala:39)
	at kafka.Kafka$.main(Kafka.scala:67)
	at kafka.Kafka.main(Kafka.scala)
```

But the zookeeper service is up & running (listening in port 2181):

```bash
vagrant@zookeeper:~$ netstat -tulpan
(Not all processes could be identified, non-owned process info
 will not be shown, you would have to be root to see it all.)
Active Internet connections (servers and established)
Proto Recv-Q Send-Q Local Address           Foreign Address         State       PID/Program name    
tcp        0      0 0.0.0.0:22              0.0.0.0:*               LISTEN      -                   
tcp        0    176 10.0.2.15:22            10.0.2.2:49603          ESTABLISHED -                   
tcp6       0      0 :::22                   :::*                    LISTEN      -                   
tcp6       0      0 :::39223                :::*                    LISTEN      -                   
tcp6       0      0 :::2181                 :::*                    LISTEN      -                   
udp        0      0 0.0.0.0:68              0.0.0.0:*                           -                   
udp        0      0 0.0.0.0:68              0.0.0.0:*                           - 
```                  

### Steps to Reproduce

Could be reproduced executing tests using _molecule test_

**Expected behavior:** Tests passing

**Actual behavior:** Tests don't pass

**Reproduces how often:** 100%

### Versions

1.3.x

### Additional Information

**Ansible version:** 2.4.2.0
**Vagrant version:** 1.8.6
**Molecule version:** 1.25.0
**Testinfra version:** 1.11.0

**Comment by dortegau** [comment]
I think that could happen because the zookeeper service is exposed using _tcp6_ not _tcp_

**Comment by eskabetxe** [comment]
resolved on #20 

working with 
  - pip install ansible==2.4.0.0
  - pip install molecule==1.25.0
  - pip install ansible-lint==3.4.20
  - pip install testinfra==1.7.1
